critiquebrainz/recipes/install.rb

```diff
@@ -0,0 +1,54 @@
+apt_package [
+  "memcached",
+]
+
+
+include_recipe "critiquebrainz::user"
+
+project_home = node['critiquebrainz']['home'] + '/critiquebrainz'
+
+include_recipe 'git::default'
+git project_home do
+  repository "git://github.com/metabrainz/critiquebrainz.git"
+  action :sync
+  group node['critiquebrainz']['group']
+  user node['critiquebrainz']['user']
+end
+
+
+# Database
+include_recipe 'postgresql::server'
+include_recipe 'postgresql::contrib'
+
+
+# Python & its packages
+python_runtime 'python' do
+  version '2.7'
+  options pip_version: true # latest
+  options virtualenv_version: true # latest
+  options wheel_version: true # latest
+end
+venv_location = project_home + '/venv'
+python_virtualenv venv_location do
+  group node['critiquebrainz']['group']
+  user node['critiquebrainz']['user']
+end
+pip_requirements project_home + '/requirements.txt' do
+  group node['critiquebrainz']['group']
+  user node['critiquebrainz']['user']
+end
+
+
+# Node.js (Less compiler)
+include_recipe "nodejs"
+nodejs_npm "less"
+nodejs_npm "less-plugin-clean-css"
```

Comment:
I'd make sure to specify a version for these (less-plugin-clean-css is even one of the few modules I remember breaking for MBS :)). Looks like you can do that like https://github.com/redguide/nodejs/blob/2b992fca4c6b7b475fcc53c3a19b0057ed204760/test/cookbooks/nodejs_test/recipes/npm.rb#L12 or can alternatively use npm's shrinkwrap stuff and run npm install manually, like MBS does.